README.md

```diff
@@ -3,8 +3,8 @@
 ### 🚨🚨🚨 IMPORTANT INFORMATION ABOUT 1.20 SUPPORT 🚨🚨🚨
 
 EKS-D has discontinued support of Kubernetes v1.20. While there are no
-plans to removed EKS-D v1.20 images from the ECR, there will be no more 
-updates to v1.20. **Due to the increased security risk this poses, it is 
+plans to removed EKS-D v1.20 images from the ECR, there will be no more
+updates to v1.20. **Due to the increased security risk this poses, it is
 HIGHLY recommended that users of v1.20 update to a supported version (v1.21+)
 as soon as possible.**
```

Comment:
We aren't supporting 1.20 anymore, right. This should be removed?

Reply:
Just trimmed two whitespaces here.